samples/bidirectional-chat/csharp/Function.cs

```diff
@@ -12,25 +12,23 @@ namespace FunctionApp
 {
     public class SimpleChat : ServerlessHub
     {
-        private const string Hub = nameof(SimpleChat);
         private const string NewMessageTarget = "newMessage";
         private const string NewConnectionTarget = "newConnection";
 
         [FunctionName("negotiate")]
-        public static SignalRConnectionInfo GetSignalRInfo(
-            [HttpTrigger(AuthorizationLevel.Anonymous)] HttpRequest req,
-            [SignalRConnectionInfo(HubName = Hub, UserId = "{headers.x-ms-signalr-user-id}")] SignalRConnectionInfo connectionInfo)
+        public SignalRConnectionInfo Negotiate([HttpTrigger(AuthorizationLevel.Anonymous)]HttpRequest req)
         {
-            return connectionInfo;
+            return Negotiate(req.Headers["x-ms-signalr-user-id"], GetClaims(req.Headers["Authorization"]));
         }
 
-        [FunctionName(nameof(Connected))]
-        public async Task Connected([SignalRTrigger]InvocationContext invocationContext, ILogger logger)
+        [FunctionName(nameof(OnConnected))]
```

Comment:
function name should be constant

Reply:
Method name in class based model has special meanings. When method name changed, function name changes make sense. Constant is OK but `nameof` is also acceptable.